test/testng/org/openjdk/jextract/test/toolprovider/Test8245767.java

```diff
@@ -44,10 +42,6 @@ public void testTypedefs() {
             Class<?> cls = loader.loadClass("test8245767_h");
             assertNotNull(cls);
 
-            // class should be generated for typedef on opaque struct
-            Class<?> fooCls = loader.loadClass("Foo");
-            assertNotNull(fooCls);
-
```

Comment:
This is the problematic test case that I believe to be testing for behavior that we do not want.

Reply:
I've seen that yesterday, and was equally puzzled by the "expected behavior"